Source/Core/DolphinQt/Debugger/WatchWidget.cpp

```diff
@@ -178,14 +181,15 @@ void WatchWidget::Update()
     m_table->setItem(i, 2, hex);
     m_table->setItem(i, 3, decimal);
     m_table->setItem(i, 4, string);
+    m_table->setItem(i, 5, floatValue);
   }
 
   auto* label = new QTableWidgetItem;
   label->setData(Qt::UserRole, -1);
 
   m_table->setItem(size, 0, label);
 
-  for (int i = 1; i < 5; i++)
+  for (int i = 1; i < 6; i++)
```

Comment:
Can we `constexpr` this number and also use it up there with `setColumnCount`?
Or, if we have a better alternative (like an `array` or whatever) that returns us that exact number; use that instead please.

Reply:
There isn't currently an `array`, beyond the temporary one used earlier:

https://github.com/dolphin-emu/dolphin/blob/58c78a495d783de3cb6928abd0c7810777decfc7/Source/Core/DolphinQt/Debugger/WatchWidget.cpp#L168

Reply:
I've attempted to do this.  I'm not super skilled with C++ so I'm not sure I did it the best way.